### DISCUSSION TITLE
Could some of the command line options be incorporated into the UI?

### DISCUSSION THREAD

**Discussion by stevie2sheds** [jamulussoftware/jamulus#3291] [UI Related]
It would be useful to have the command line options within UI.

Since moving to an ISP with CGNAT I can get better routes, and therfore latency, using IPv6 so use the -6 option. I can also host a private server at home on IPv6 no problem. However we have have clients on Linux, Windows, and MacOS, and starting the Jamulus client on Macos at the command line is the particular nuisance for the less technically minded. It would be easier to have a check box within the UI or even have IPv6 enabled by default so the option wouldn't even be required.

Other options could also be within the UI such as the default server and port you wish to connect to.

**Comment by ann0see** [comment]
Yes, there's ongoing effort on IPv6 as it solves some other issues also. I believe adding a checkbox was discussed in the past but never implemented as IPv6 support is still considered not completed.

**Reply by stevie2sheds** [reply]
Thanks for the information. Yes, ideally a fully completed and implemented ipv6 negates the need for a checkbox.

It's interesting to learn that the IPv6 support isn't considered complete. What are the current issues? I have found that a dual stack server running for 2.5 hours with 5 clients can suddenly get network issues with huge delays, but I hadn't investigated why (we only play for 2 hours!) I had put it down to being free tier AWS and the IPv4 addressing not staying stable rather than the IPv6, but that is only speculation.

**Reply by ann0see** [reply]
Directories don't yet work on IPv6 but @softins is working on it.

**Reply by ann0see** [reply]
Can you open an issue to ask for explicitly enabling ipv6 support client side by default? I think this is doable as we didn't see issues with this feature.

**Reply by stevie2sheds** [reply]
Yes, I'll do that.

**Comment by pljones** [comment]
#1973 covers UI/inifile for the flag.
